typescript/cli/src/tests/cosmosnative/commands/core.ts

```diff
@@ -0,0 +1,160 @@
+import { $, ProcessPromise } from 'zx';
```

Comment:
Condiering that this file is mostly the same as the evm counterpart, can't we just update the already existing functions to accepts cosmos related values instead? so we avoid duplication?

Reply:
the remaining evm helpers like deploy, read, apply will be removed once the warp tests have been refactored in the next PR